module/metrics/verification.go

```diff
@@ -2,15 +2,17 @@ package metrics
 
 import (
 	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/promauto"
-	"github.com/rs/zerolog"
 
 	"github.com/onflow/flow-go/module"
 )
 
 type VerificationCollector struct {
-	tracer          module.Tracer
-	storagePerChunk prometheus.Gauge // storage per chunk
+	tracer module.Tracer
+
+	// Assigner Engine
+	rcvBlocksTotal      prometheus.Counter // total finalized blocks received by assigner engine
+	assignedChunksTotal prometheus.Counter // total chunks assigned to this verification node
+	sntChunksTotal      prometheus.Counter // total chunks sent by assigner engine to chunk consumer (i.e., fetcher input)
```

Comment:
```suggestion
	sentChunksTotal      prometheus.Counter // total chunks sent by assigner engine to chunk consumer (i.e., fetcher input)
```

I feel like we can afford the extra character 😸

Reply:
This was a common naming pattern in `verification` metrics. To address this comment, refactored them all https://github.com/onflow/flow-go/pull/466/commits/d98381102941a8fa7bb3e42af1722bf068d5ba98